.idea/workspace.xml

```diff
@@ -4,7 +4,9 @@
     <option name="autoReloadType" value="SELECTIVE" />
   </component>
   <component name="ChangeListManager">
-    <list default="true" id="2008a514-c0bf-445a-a873-e02f60f249a5" name="Changes" comment="" />
+    <list default="true" id="2008a514-c0bf-445a-a873-e02f60f249a5" name="Changes" comment="">
```

Comment:
please get rid of this file and remember to add the whole .idea folder to .gitignore